content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -396,8 +396,8 @@ kubeadm to tell it what to do.
 ## Configuring a cgroup driver
 
 Both the container runtime and the kubelet have a property called
-["cgroup driver"](/docs/setup/production-environment/container-runtimes/), which is important
-for the management of cgroups on Linux machines.
+["cgroup driver"](/docs/setup/production-environment/container-runtimes/#cgroup-drivers), which is important for the management 
+of cgroups on Linux machines.
```

Comment:
nit: Would be nice to restrict the changes to just the link and keep the rest of the lines unchanged.